internal/cuetxtar/cuetxtar.go

```diff
@@ -83,6 +85,8 @@ type Test struct {
 	Dir string
 
 	hasGold bool
+
+	ImportMapperFn func(s string) (string, error)
```

Comment:
ugh, feels like this should be something the test itself can inject, don't need it in the framework

Reply:
Yes... I neither like it. 

I added because the logic per test its inside the framework but it doesn't have any sense to have it here.

Reply:
Just removed :). 

I propagate the test name into the framework to map it into the test. I needed something to know what is the test that I wanted to apply the imports.